projects/solar/test/test_sense_temperature.c

```diff
@@ -12,6 +12,11 @@
 
 #define TEST_ADC_RAW_READING 0x800
```

Comment:
This is actually the `TEST_ADC_CONVERTED_READING` now eh? And for converted readings it's more idiomatic to give it in decimal since it's an actual value in mV (in this case 2048mV).

Reply:
I'm a bit confused on what the change requested here is.

Reply:
Ah sorry, what I meant was change this variable to `TEST_ADC_CONVERTED_READING` since it's returned from `adc_read_converted` now, not `adc_read_raw`, and specify its value in decimal rather than hex because it's an actual value in mV.